Core.ApplicationServices/UIConfiguration/UIModuleCustomizationService.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Core.Abstractions.Types;
+using Core.ApplicationServices.Authorization;
+using Core.ApplicationServices.Organizations;
+using Core.DomainModel.Organization;
+using Core.DomainModel.UIConfiguration;
+using Infrastructure.Services.DataAccess;
+
+namespace Core.ApplicationServices.UIConfiguration
+{
+    public class UIModuleCustomizationService : IUIModuleCustomizationService
+    {
+        private readonly ITransactionManager _transactionManager;
+        private readonly IAuthorizationContext _authorizationContext;
+        private readonly IOrganizationalUserContext _userContext;
+        private readonly IOrganizationService _organizationService;
+
+        public UIModuleCustomizationService(ITransactionManager transactionManager,
+            IAuthorizationContext authorizationContext,
+            IOrganizationalUserContext userContext,
+            IOrganizationService organizationService)
+        {
+            _transactionManager = transactionManager;
+            _authorizationContext = authorizationContext;
+            _userContext = userContext;
+            _organizationService = organizationService;
+        }
+
+
+        public Result<List<UIModuleCustomization>, OperationError> GetModuleConfigurationForOrganization(int organizationId, string module)
+        {
+            var organization = GetOrganizationById(organizationId);
+            if (organization == null || organization.UIModuleCustomizations == null)
+                return new OperationError(OperationFailure.NotFound);
+            if (!_authorizationContext.AllowReads(organization))
```

Comment:
When testing I can't get the AllowReads method to return false, even if user is from another organization